tests/test_exporter.py

```diff
@@ -98,7 +98,7 @@ def case(self, test, files, *pages):
             if not os.path.exists(expected_file):
                 expected_file = file(f'test{test}_out')
 
-        export(files, pages, [], [file('out')], None, None, True)
+        export(files, pages, {}, [file("out")], None, None, True)
```

Comment:
Array is incorrect because we get

```
  File "pdfarranger/pdfarranger/metadata.py", line 106, in merge_doc
    r[k] = v
TypeError: list indices must be integers or slices, not str
```